tetrateci/patches/eks/eks-ingress.1.13.patch

```diff
@@ -0,0 +1,61 @@
+diff --git a/pkg/test/framework/components/echo/kube/deployment.go b/pkg/test/framework/components/echo/kube/deployment.go
```

Comment:
How is the patch generated? How will these help the test?

Reply:
I saw a failure in the CI run, https://github.com/tetratelabs/istio/runs/5880304061?check_suite_focus=true in the eks_e2e_test.

```
2022-04-08T10:10:52.0508892Z 2022-04-08T10:10:52.050637Z	warn	tf	failed to get address for port 80: service istio-system/istio-ingressgateway is not available yet: no ingress
```

Is that what this patch trying to solve?

Reply:
This patch is backported from earlier releases.
For new release we can run `git apply --3way eks_ingress tetrateci/patches/eks/eks-ingress.1.13.patch` > newpatch

Reply:
@incfly :  regarding failure in CI Run,
`failed to get address for port 80: service istio-system/istio-ingressgateway `
i think this is due to delay in ingress endpoint availabilty, once ip/hostname is assigned by LB it goes away.

Reply:
Thanks. Now I understand that the patch are just copied from previous release. Might worth documenting somewhere in our `docs/*.md`.